### DISCUSSION TITLE
License application -- Requirement for annual accounts of 10%+ shareholders

### DISCUSSION THREAD

**Discussion by kennethng-ledn** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#39] [General]
Regarding Section A.2 of the proposed Regulations, this requirement would be very difficult for VASPs with private equity/venture capital investors to comply with.  It is common for PE/VC investors to have a 10%+ position, and such investors are generally unwilling to disclose their financial position to their portfolio companies.  Should this requirement come into effect, it would likely cause PE/VC investors to keep their investment under 10%, which would lead to reduced access to capital for VASPs. We see this similar dynamic play out in the context of US banks and Bank Holding Company Act requirements. 

One suggestion is to raise this requirement to disclose annual accounts to 25%+ shareholders only.  

10%+ shareholders are already subject to the KYC/UBO requirements as part of the VASP application process.

**Comment by AGIAI** [comment]
@kennethng-ledn - I don't even believe this makes sense at 25% - most PE firms will take a majority position; it seems an unreasonable ask when compared to tradition finance legislation in Cayman

**Comment by FinalTwist** [comment]
How often would a VC investor take an equity position in a project?  I understand that most VC currently take early distribution of tokens, etc as a means of future compensation?

These requirements are likely there for reason of the beneficial ownership regime here in Cayman.

**Reply by kennethng-ledn** [reply]
@FinalTwist I think you're right with respect to the decentralized projects.  With respect to the many VASPs in the industry that are non-DeFi related, centralized "CeFi" businesses, it is very common for a VC to have an equity position. 

The UBO regime is still applicable to such VC investors, which will be required to provide all the customary KYC/KYB documents. However, the requirement to disclose accounts is a major step above that, and having personally dealt with virtual asset VCs over the past 5 years, I can say that it is a non-starter in many cases.  In my view this would cause such investors to make their investments at no more than 9.9%, which would harm capital formation in this space.

**Comment by CaraProvenance** [comment]
@kennethng-ledn  a very valid point i.e. that VC Investors, perceived data risk, may influence their level of participation, and investment in Cayman VASPs.  Looking through the Compliance lens, I am assuming this requirement arises from the elevated risk profile that virtual assets are categorized under, which in turn results in a requirement to conduct enhanced rather than standard due diligence. 

However, it would be worth exploring alternative methods of complying with the requirement to prove source and substance funding/wealth of the Capital behind the applicant.

If Shareholders meet certain criteria for e.g. fall in the category of regulated VCs, as opposed to Individual or Family Office Investors and  have completed full KYC/KYB, with nil red flags emerging and there is also some other form of financial/banking reference that should provide a sound, and appropriate level of due diligence.

Also worth comparing what other jurisdictions are requesting on applications re shareholders financial records.
